src/app/zeko/default.nix

```diff
@@ -1,28 +0,0 @@
-inputs: system:
```

Comment:
You have deleted this file, which makes sense, but not the references to it I think